src/app/Dependencies.tsx

```diff
@@ -0,0 +1,124 @@
+import { useState, useCallback, useEffect } from 'react';
+import { jsx } from '@emotion/core';
+import ReactFlow, { ReactFlowProvider, addEdge, removeElements, isNode } from 'react-flow-renderer';
+import * as dagre from 'dagre';
+import { v4 as uuid } from 'uuid';
+import { getDependencyMap } from './commands';
+import { useStore } from './store';
+
+export const Dependencies = () => {
+  const position = { x: 0, y: 0 };
+  const edgeType = 'smoothstep';
+  const test = useStore(m => m.state.dependencyMap);
+
+  const dependencies = {
```

Comment:
Don't use the hardcoded one - we are interested in the "live" one (i.e., from the store).